diff --git a/node_modules/mitata/src/cli.d.ts b/node_modules/mitata/src/cli.d.ts
index acd42b3..415130f 100644
--- a/node_modules/mitata/src/cli.d.ts
+++ b/node_modules/mitata/src/cli.d.ts
@@ -5,6 +5,8 @@ export function group(fn: () => void): void;
 export function group(name: string, fn: () => void): void;
 export function group(options: { name?: string, summary?: boolean }, fn: () => void): void;
 
+export function prepare(name: string, fn: () => any): void;
+
 export function run(options?: {
   avg?: boolean,
   silent?: boolean,
diff --git a/node_modules/mitata/src/cli.mjs b/node_modules/mitata/src/cli.mjs
index 45be23e..651a206 100644
--- a/node_modules/mitata/src/cli.mjs
+++ b/node_modules/mitata/src/cli.mjs
@@ -32,6 +32,7 @@ export function bench(name, fn) {
     time: 500,
     warmup: true,
     baseline: false,
+    hook: false,
     async: AsyncFunction === fn.constructor,
   });
 };
@@ -47,10 +48,27 @@ export function baseline(name, fn) {
     time: 500,
     warmup: true,
     baseline: true,
+    hook: false,
     async: AsyncFunction === fn.constructor,
   });
 };
 
+export function prepare(name, fn) {
+  if ([Function, AsyncFunction].includes(name.constructor)) (fn = name, name = fn.name);
+  if (![Function, AsyncFunction].includes(fn.constructor)) throw new TypeError(`expected function, got ${fn.constructor.name}`);
+  
+  benchmarks.push({
+    fn,
+    name,
+    group: g,
+    time: 500,
+    warmup: false,
+    baseline: false,
+    hook: true,
+    async: AsyncFunction === fn.constructor,
+  });
+}
+
 let _print;
 
 try {
@@ -243,6 +261,11 @@ export async function run(opts = {}) {
       _f = true;
 
       try {
+        if (b.hook) {
+          await b.fn();
+          continue;
+        }
+
         b.stats = (await measure(b.fn, {})).stats;
         if (!json) log(table.benchmark(b.name, b.stats, opts));
       }
@@ -267,6 +290,11 @@ export async function run(opts = {}) {
         if (group !== b.group) continue;
 
         try {
+          if (b.hook) {
+            await b.fn();
+            continue;
+          }
+
           b.stats = (await measure(b.fn, {})).stats;
           if (!json) log(table.benchmark(b.name, b.stats, opts));
         }
